application-form/src/components/admin/AdminHome.vue

```diff
@@ -0,0 +1,158 @@
+<template>
+  <div class="admin-page">
+    <SideBar @handleLogout="handleLogout" :user="userDetail" />
+    <div class="main-content">
+      <div class="table-container">
+        <table class="table-item" cellspacing="0">
+          <tr class="table-header">
+            <th v-for="(item, index) in header" :key="index">
+              <p>{{ item }}</p>
+            </th>
+          </tr>
+          <UserList
+            v-for="form in filterdForm"
+            :key="form.id"
+            :form="form"
+            @onDetailUser="onDetailUser(form.id)"
+          />
+        </table>
+      </div>
+      <FooterTable
+        @handleBack="handleBack"
+        @handleNext="handleNext"
+        :currentPage="currentPage"
+        :start="start"
+        :end="end"
+      />
+    </div>
+  </div>
+</template>
+
+<script>
+import UserList from "./components/UserList.vue";
+import FooterTable from "./components/FooterTable.vue";
+import SideBar from "./components/SideBar.vue";
+import { mapActions, mapGetters } from "vuex";
+import { USER_PER_PAGE } from "@/const/multiForm.js";
+export default {
+  data() {
+    return {
+      currentPage: 1,
+      start: 0,
+      end: 0,
+      header: ["Fullname", "City", "Wish Salary", "Created at", "Status"],
+    };
+  },
+  components: {
+    SideBar,
+    UserList,
+    FooterTable,
+  },
+  created() {
+    this.getUser();
+  },
+  methods: {
+    ...mapActions("form", ["getUser", "clearForm"]),
+    ...mapActions("list", ["clearChoseList"]),
+    ...mapActions("loading", ["activeLoading"]),
+    handleLogout() {
+      this.clearForm();
+      this.clearChoseList();
+      this.activeLoading();
+      setTimeout(() => {
+        this.$router.push("/");
+      }, 2000);
+    },
+    onDetailUser(id) {
+      this.$router.push({ name: "/admin/detail", params: { id } });
+    },
+    handleNext() {
+      this.currentPage++;
+    },
+    handleBack() {
+      this.currentPage--;
+    },
+    filteredList() {
+      let start = (this.currentPage - 1) * USER_PER_PAGE;
+      let end = start + USER_PER_PAGE;
+      this.start = start + 1;
```

Comment:
sao lại gán cái this.start trong một cái hàm mà cái hàm này lại là giá trị trả về của một cái computed nhỉ?